src/swagger-specs/query-service.yaml

```diff
@@ -1825,4 +2174,544 @@ definitions:
+    type: object
+    description: The error response for a 404 status when requesting subscriber information for an asset ID and alert type.
+    properties:
+      message:
+        type: string
+        description: A description of the error to explain the problem and instruct how to resolve the issue. 
+        example: 'No alert exists for assetId: c4f67291-1161-4943-bc29-8736469bb928sa and alertType: failure. Please resubmit with different assetId and alertType.'
+      statusCode:
+        $ref: '#/definitions/NotFoundError'
+
+  get_alert_subscription_for_asset_id_and_alert_type_bad_request_error_message:
+    type: object
+    description: The error response for a 400 status when requesting subscriber information for an asset ID and alert type.
+    properties:
+      message:
+        type: string
+        description: A description of the error to explain the problem and instruct how to resolve the issue. 
+        example: 'An invalid alertType was provided. Valid values are: start, success and failure.'
+      statusCode:
+        $ref: '#/definitions/BadRequestError'
+
+
+  post_bad_request_error_message:
+    type: object
+    description: The error response for a 400 status for a request to create and subscribe to a new alert.
+    properties: 
+      message:
+        type: string
+        description: A description of the error.
+        example: 'The provided emailId: scoobydoo@adobe.com is not an Adobe registered ID. Please use another emailId.'
+      statusCode:
+        $ref: '#/definitions/BadRequestError'
+
+  post_not_found_error_message:
+    type: object
+    description: The error response for a 404 status for a request to create and subscribe to a new alert.
+    properties: 
+      message:
+        type: string
+        description: A description of the error.
+        example: 'The provided assetId: 24a218c7-1738-4f56-95fb-asda62928751a680 is invalid. Retry with another assetId.'
+      statusCode:
+        $ref: '#/definitions/NotFoundError'
+
+  Id:
+    type: string
+    description: The alert name that appears in the alerts tab of the Adobe Experience Platform UI.
+    example: 'query_service_flow_run_failure-5cdc3bbe-750a-4d80-9c43-96e5e09f1a96'    
+    
+  AlertType:
+    description: This property determines the state of query execution that triggers a user to receive an alert. The alert type is one of three values (start, success, failure).
+    type: string
+    example: 'failure' 
+  
+  Status:
+    description: This represents the status of the alert. It is one of four values (enabled, enabling, disabled, disabling).
+    type: string
+    example: 'enabled' 
+  
+  AssetId:
+    description: The query or schedule ID that the alert is created for and subscribed to.
+    type: string
+    example: '6df22232-f427-4250-a4e1-43cd30990641' 
+
+  AlertName:
+    description: The name of the alert. This name is generated by the Alerts service and used on the Platform Alerts dashboard.
```

Comment:
Is "Alerts service" a public-facing term? Do we know the capitalization rules around it? Same thing re: "Platform Alerts dashboard"...